cmd/ignite/run/create_test.go

```diff
@@ -120,53 +251,28 @@ func TestConstructVMFromCLI(t *testing.T) {
 					PublicKey: "some-pub-key",
 				},
 			},
-			args: []string{testImage},
 			wantSSH: api.SSH{
 				Generate:  true,
 				PublicKey: "some-pub-key",
 			},
 		},
-		{
-			name: "with no VM name and --require-name flag set",
-			createFlag: &CreateFlags{
-				VM:          &api.VM{},
-				RequireName: true,
-			},
-			args: []string{testImage},
-			err:  true,
-		},
-		{
-			name: "with sandbox image",
-			createFlag: &CreateFlags{
-				VM: &api.VM{
-					Spec: api.VMSpec{
-						Sandbox: api.VMSandboxSpec{
-							OCI: sandboxOCIRef,
-						},
-					},
-				},
-			},
-			args:             []string{testImage},
-			wantSandboxImage: sandboxOCIRef,
-		},
 	}
 
 	for _, rt := range tests {
 		t.Run(rt.name, func(t *testing.T) {
-			err := rt.createFlag.constructVMFromCLI(rt.args)
+			vm := rt.createFlag.VM
+			fs := flag.NewFlagSet("test", flag.ExitOnError)
```

Comment:
This does not test flag validity since that would need the flag set to be populated (needs additional logic), but the flag handling will need to be refactored at some point anyway so this is fine.